botorch/acquisition/acquisition.py

```diff
@@ -64,6 +66,37 @@ def forward(self, X: Tensor) -> Tensor:
         """
         pass  # pragma: no cover
 
+    @classmethod
+    def construct_inputs(
+        cls,
+        model: Model,
+        training_data: TrainingData,
+        objective: Optional[AcquisitionObjective] = None,
+        **kwargs: Any
+    ) -> Dict[str, Any]:
+        r"""Construct kwargs for the `AcquisitionFunction` constructor.
+
+        Args:
+            model: The model to be used in the acquisition function.
+            training_data: A TrainingData object contraining the model's
+                training data. Used e.g. to extract inputs such as `best_f`
+                for expected improvement acquisition functions.
+            objective: The objective to in the acquisition function.
```

Comment:
Nice feature! Just two typos - "contraining" & "the objective to in". These repeat in all `construct_inputs` docstrings.